gitlab/mixins.py

```diff
@@ -477,9 +476,7 @@ def delete(self, id: Union[str, int], **kwargs: Any) -> None:
         if id is None:
             path = self.path
         else:
-            if not isinstance(id, int):
-                id = utils._url_encode(id)
-            path = f"{self.path}/{id}"
+            path = f"{self.path}/{utils._url_encode(id)}"
```

Comment:
Same question as above. Mind you I didn't check in detail :laughing:

Reply:
In this case `id` is not from `self.id`.